File: app/src/main/java/dagger/nambv/app/demo/presenter/OnPromotionsCompletedListener.java
package dagger.nambv.app.demo.presenter;

import java.util.List;

import dagger.nambv.app.demo.model.Repo;

public interface OnPromotionsCompletedListener {
    void onRepoListRetrievalSuccess(List<Repo> data);
    void onRepoListRetrievalFailure(String errorMessage);
}

File: app/src/main/java/dagger/nambv/app/demo/presenter/OnPromotionsCompletedListener.kt
package dagger.nambv.app.demo.presenter

import dagger.nambv.app.demo.model.Repo

interface OnPromotionsCompletedListener {
    fun onRepoListRetrievalSuccess(data: List<Repo>)
    fun onRepoListRetrievalFailure(errorMessage: String)
}
